ui/app/styles/core.scss

```diff
@@ -62,6 +62,7 @@
 @import './components/confirm';
 @import './components/console-ui-panel';
 @import './components/control-group';
+@import './components/dashboard';
```

Comment:
so technically these are only meant to correspond to a component called "dashboard". And it looks like you're using a route called Dashboard. 

Maybe you and I can sync real quick on this. Ideally (I know you mentioned above about it not being a helper class) but it really doesn't make sense as a component class.

Reply:
synced with angel and will move dashboard route template to a component file if extra styling is needed!